app/serializers/compliance_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# JSON serializer parent class
+module ComplianceSerializer
+  extend ActiveSupport::Concern
+  include ::JSONAPI::Serializer
+
+  class_methods do
+    def relationships_hash(record, _, _, _, params = {})
+      return {} if params[:root_resource] != record.class
+
+      super
+    end
```

Comment:
Maybe we should add a comment about this being hacky and dependent on jsonapi_serializer's implementation. It seems like it'd be pretty stable, though, as long as the method name doesn't change :see_no_evil: